docs/ntppool/use/sample-legacy-config.html

```diff
@@ -1,8 +1,8 @@
 <pre class="code">
 driftfile /var/lib/ntp/ntp.drift
 
-server 0.pool.ntp.org
-server 1.pool.ntp.org
 server 2.pool.ntp.org
 server 3.pool.ntp.org
+server 0.pool.ntp.org
+server 1.pool.ntp.org
```

Comment:
Why is the order changed here? I'd not expect that to make a difference and it just looks odd.

Reply:
I agree - there doesn't seem much value in changing this, especially given that most distributions give it in numeric order, and this would lead users to think that there's some significance in the change.

Reply:
It's already been mentioned elsewhere, but closure's sake, the order has been changed in order to promote IPv6 pool servers and 2.pool is the only pool containing IPv6 entries.